### ISSUE TITLE
Info menu: clicking the publisher name should show packages from that publisher

### ISSUE THREAD

**Issue by ppvnf** [martinet101/WingetUI#219]
For example, if you search https://winget.run/search?query=WingetUI on winget.run and then click Martí Climent it's possible to see all your apps. 

Currently on WingetUI it's possible to do something similar, but it's a little bit more difficult: you have to copy "SomePythonThings." and paste it on the search bar.

So, I believe it would be easier if I could just click the publisher's name instead of doing the above:

![Captura de tela 2022-10-08 105412](https://user-images.githubusercontent.com/73800734/194711042-7eb7ccdf-180a-497c-9df1-0271b4d00d53.png)

**Comment by martinet101** [comment]
You mean like directly searching for the name in the search box, right?

**Comment by ppvnf** [comment]
> You mean like directly searching for the name in the search box, right?

Yes, something like this. Ideally the publisher's name would work as a hyperlink that closes the info menu and searches that name on WingetUI, so the user would be able to instantly see all the packages from that publisher by simply clicking its name. However, I think this would only work on Winget as publishers' information seems absent in Scoop

**Comment by martinet101** [comment]
didn't thought about that...

**Comment by ppvnf** [comment]
> didn't thought about that...

The downside of my idea is that the user would see "SomePythonThings." in the search bar, instead of Martí Climent, so maybe there is room for improvement

**Comment by martinet101** [comment]
> > didn't thought about that...
> 
> The downside of my idea is that the user would see "SomePythonThings." in the search bar, instead of Martí Climent, so maybe there is room for improvement



Well, that's my case cause I'm dumb and wrote a random ID.

Normal software publishers use their own name in their IDs.


P. D. I'm trying to change this and have the ID be MartiCliment.WingetUI

**Comment by ppvnf** [comment]
> Well, that's my case cause I'm dumb and wrote a random ID.
> 
> Normal software publishers use their own name in their IDs.
> 
> P. D. I'm trying to change this and have the ID be MartiCliment.WingetUI

IMAO. I've also done this with my first e-mail... Good luck! 

I've found another one: "brrd.abricotine". Anyway, how about creating an alias for the publisher's id? Like making "Publisher: Thomas Brouard" display the same search reasult as "brrd." would do?

**Comment by martinet101** [comment]
Yes, I think this makes sense
